Makefile

```diff
@@ -122,11 +124,16 @@ build-itest-race:
 	@$(call print, "Building itest binary for ${backend} backend.")
 	CGO_ENABLED=0 $(GOTEST) -v ./itest -tags="$(DEV_TAGS) $(RPC_TAGS) integration $(backend)" -c -o itest/itest.test$(EXEC_SUFFIX)
 
-#? install: Build and install lnd and lncli binaries, place them in $GOPATH/bin
+#? install: Build and install lnd and lncli binaries, place them in $GOPATH/bin, generate and install man pages.
 install:
 	@$(call print, "Installing lnd and lncli.")
 	$(GOINSTALL) -tags="${tags}" -ldflags="$(RELEASE_LDFLAGS)" $(PKG)/cmd/lnd
 	$(GOINSTALL) -tags="${tags}" -ldflags="$(RELEASE_LDFLAGS)" $(PKG)/cmd/lncli
+	make manpages
+
+manpages:
```

Comment:
I think calling `make` from within `make` is a bit weird. How about we do it this way instead to have proper dependencies between build goals:
```
#? install-binaries: Build and install lnd and lncli binaries, place them in $GOPATH/bin.
install-binaries:
	@$(call print, "Installing lnd and lncli.")
	$(GOINSTALL) -tags="${tags}" -ldflags="$(RELEASE_LDFLAGS)" $(PKG)/cmd/lnd
	$(GOINSTALL) -tags="${tags}" -ldflags="$(RELEASE_LDFLAGS)" $(PKG)/cmd/lncli

#? manpages: generate and install man pages.
manpages:
	@$(call print, "Generating man pages lncli.1 and lnd.1.")
	./scripts/gen_man_pages.sh $(DESTDIR) $(PREFIX)

#? install: Build and install lnd and lncli binaries, place them in $GOPATH/bin, generate and install man pages.
install: install-binaries manpages
```

Reply:
Very nice! thanks for this also thanks for adding the help comment above the make target forgot to added it 👍